controllers/comments.controllers.js

```diff
@@ -4,15 +4,18 @@ const {selectArticleById} = require('../models/articles.models')
 exports.sendCommentsByArticleId = (req, res, next) => {
 
     const {article_id} = req.params;
+    const promises = [selectArticleById(article_id)]
 
-    selectArticleById(article_id)
-    .then(() => {
-        return selectCommentsByArticleId(article_id)
+    if(article_id){
+        promises.push(selectCommentsByArticleId(article_id))
```

Comment:
Nice use of Promise.all!  I would remove (line 9 -11) the step of checking if(article_id) and pushing the promises array.   You can just put the selectCommentsByArticleID function in the promise array to start with.  You are getting comments by article ID. If article_id is undefined, you'll just get a psql error that your error handlers will get anyway anyway!  This function should not work without an article_id